docs/tutorial/installation.md

```diff
@@ -69,6 +69,11 @@ For instance, to use the China mirror:
 ELECTRON_MIRROR="https://cdn.npm.taobao.org/dist/electron/"
 ```
 
+By default, `ELECTRON_CUSTOM_DIR` is set to `v$VERSION`. To change the format,
+use the `{{ version }}` placeholder. For example, `version-{{ version }}`
```

Comment:
> To change the format, use the `{{ version }}` placeholder

Nit: As someone who hasn't used `@electron/get` before, this phrasing is a bit vague to me. Does it mean that to set the dir to `version-5.0.0`, you should:

1) `ELECTRON_CUSTOM_DIR="version-{{ version }}"`
2) `url = ELECTRON_MIRROR + version-{{ version }} + '/' + ELECTRON_CUSTOM_FILENAME`
3) Either?

I'm leaning more towards #1 from what I'm understanding, but I think making it explicit would help.

Reply:
It's 1. Option 2 is kind of implied. I'll see if I can make it clearer.

Reply:
My attempt at clarity is in https://github.com/electron/electron/pull/22491/commits/520dcd2780412df91ce8c4ce2794ec845f6a3916

Reply:
:shipit: